manifests/p/Piriform/CCleaner/6.25.11131/Piriform.CCleaner.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Piriform.CCleaner
+PackageVersion: 6.25.11131
+PackageLocale: en-US
+Publisher: Piriform
+PublisherUrl: https://www.ccleaner.com/
+PublisherSupportUrl: https://support.ccleaner.com/
+PrivacyUrl: https://www.ccleaner.com/legal/general-privacy-policy
+Author: Piriform Software Ltd
+PackageName: CCleaner
+PackageUrl: https://www.ccleaner.com/ccleaner
+License: Freeware
+LicenseUrl: https://www.ccleaner.com/legal/end-user-license-agreement
+Copyright: Copyright © 2005-2024 Gen Digital Inc. All rights reserved.
+CopyrightUrl: https://www.ccleaner.com/about/terms-of-use
+ShortDescription: CCleaner for PC helps you clean up your Windows computer, optimize performance, and maintain your online privacy.
+Moniker: ccleaner-free
+Tags:
+- cleaner
+- cleanup
+- free
+- optimize
+- performance
+- registry
+- startup
+- updater

```

Comment:
Add ReleaseNotes

```suggestion
- updater
ReleaseNotes: |-
  Taking the hassle out of PC maintenance
  - The Driver Updater database has now been expanded to contain more than 90 million drivers to help maintain optimal PC performance.
  - You are now able to cancel the downloading of driver updates currently in progress.
  - We've updated the Software Updater database to the latest version so that it includes additional apps and their updates.

  Fixing bugs and other improvements
  - We fixed an issue that could have caused CCleaner to interrupt the uploading of larger files to Google Drive when running a cleanup.
  - We resolved an issue where, in certain cases, the number of drivers to update displayed on the Driver Updater icon was incorrect.
  - The "Scan Now" button on the Health Status screen now correctly triggers the Health Check scan.
  - By enabling remote log upload in the CCleaner Support Tool in the case of issues, we are now able to collect larger crash dumps that cannot be sent by email.
  - The Support Tool is now saved to a new, more suitable location.
  - We have made improvements to boost CCleaner performance and stability.
ReleaseNotesUrl: https://www.ccleaner.com/knowledge/ccleaner-v6-25-11131
```